frontend/src/features/mapSideBar/tabs/SectionField.tsx

```diff
@@ -0,0 +1,37 @@
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+
+interface ISectionFieldProps {
+  label: string;
+  content: string[];
```

Comment:
could we make this a ReactNode? that would allow me to use this component.

Reply:
or ReactNode[].

Reply:
or ReactNode[]

Reply:
Actually, it might be nice to have another version of this that just provides the Row/Col structure but allows you to pass through whatever you want via child props. I can add that myself though.

Reply:
something like this: 
```
import { Col, Row } from 'react-bootstrap';
import styled from 'styled-components';

interface ISectionFieldWrapperProps {
  label: string;
}

export const SectionFieldWrapper: React.FunctionComponent<SectionFieldWrapper> = props => {
  return (
    <Row className="pb-2">
      <Col xs="4">
        <StyledFieldLabel>{props.label}:</StyledFieldLabel>
      </Col>
      <Col>{props.children}</Col>
    </Row>
  );
};

const StyledFieldLabel = styled.div`
  font-weight: bold;
`;
```

Reply:
I also prefer the version that just enforces layout and lets the consumer decide what child to render. That way we can use other things there than raw text